client/src/components/Shop/shop.js

```diff
@@ -0,0 +1,270 @@
+import React, { Component } from "react";
+// TODO: Create products db to hold prod data
+// FIXME: import API from "../../utils/API";
+import { Link } from "react-router-dom";
+import Button from "react-bootstrap/Button";
+import MyVerticallyCenteredModal from "../Modals/index";
+
+let productList = [
+  { name: "Ice Cream", price: 1, info: "Let's get some ice cream!" },
+  {
+    name: "Popcorn & Movie",
+    price: 10,
+    info: "Ready to watch a movie? Popcorn included!"
+  }
+];
+
+/* Product */
+class Product extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      modalShow: false,
+      qty: 0
+    };
+    this.add = this.add.bind(this);
+    this.subtract = this.subtract.bind(this);
+    this.showInfo = this.showInfo.bind(this);
+  }
+
+  add() {
+    this.setState({
+      qty: this.state.qty + 1
+    });
+    this.props.handleTotal(this.props.price);
+  }
+
+  subtract() {
+    this.setState({
+      qty: this.state.qty - 1
+    });
+    this.props.handleTotal(-this.props.price);
+  }
+
+  showInfo() {
+    this.props.handleShow(this.props.info);
+  }
+
+  render() {
+    let modalClose = () => this.setState({ modalShow: false });
+
+    return (
+      <div className="container">
+        <MyVerticallyCenteredModal
+          show={this.state.modalShow}
+          text="example text goes here"
+          onHide={modalClose}
+        />
+        <div className="row form-group">
+          <div className="col-sm-10">
+            <h4>
+              {this.props.name}: ${this.props.price}
+            </h4>
+          </div>
+          <div className="col-sm-2 text-right">qty: {this.state.qty}</div>
+        </div>
+        <div className="row btn-toolbar">
```

Comment:
this can be another component button type. A specific button with certain styling. extract this out.